package.json

```diff
@@ -81,9 +81,9 @@
         "title": "Sort lines (remove duplicate lines)"
       },
       {
-				"command": "sortLines.keepOnlyDuplicateLines",
-				"title": "Sort lines (keep only duplicated lines)"
-			},
+        "command": "sortLines.keepOnlyNotDuplicateLines",
+        "title": "Sort lines (keep only not duplicated lines)"
+      },
       {
```

Comment:
Looks like you deleted the old command

Reply:
Oops. Fixed on [10ac80a](https://github.com/Tyriar/vscode-sort-lines/pull/132/commits/10ac80ad01af2620993155a04e03ffec2ed3c002)